x-pack/plugins/cases/public/components/create/form_context.tsx

```diff
@@ -105,10 +105,16 @@ export const FormContext: React.FC<Props> = ({
       }
 
       for (const [key, value] of Object.entries(customFields)) {
+        let fieldValue = null;
         const configCustomField = customFieldsConfiguration.find((item) => item.key === key);
 
         if (configCustomField) {
-          const fieldValue = isEmpty(value) && typeof value !== 'boolean' ? null : [value];
+          if (!isEmpty(value) && typeof value === 'string') {
```

Comment:
Let's put this in a helper function so we can easily extend it when we add more field types. Also, let's add some unit tests (if we don't have any) to test that boolean and string values are working as expected.

Reply:
Nice!